### ISSUE TITLE
The new version does not have 'Prior: True' information and does not perform 'Cache Latents'

### ISSUE THREAD

**Issue by OrochiZ** [d8ahazard/sd_dreambooth_extension#335]
**Please find the following lines in the console and paste them below. _If you do not provide this information, your issue will be automatically closed._**
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: b5050ad2071644f7b4c99660dc66a8a95136102f
Dreambooth revision is a66a2d6bca536207ca3154324f73a5cd43a65cb4
[X] bitsandbytes version 0.35.0 installed.
[X] diffusers version 0.8.1 installed.
[X] transformers version 4.21.0 installed.
[X] torch version 1.12.1+cu116 installed.
[X] torchvision version 0.13.1+cu116 installed.
[*] xformers version 0.0.14.dev0 installed.

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes

**Have you updated Dreambooth to the latest revision?**
Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes

**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.**
OK

**Describe the bug**
The new version does not have 'Prior: True' information and does not perform 'Cache Latents'
'Training settings: CPU: False Adam: True, Prec: fp16, Grad: False, TextTr: True EM: True, LR: 5e-07'
When 'Don't Cache Latents' is not checked and setting 'Total Number of Class/Reg Images' to 1500, the 'Cache Latents' stage still does not appear.

**Environment**

What OS?
Windows 10

If Windows - WSL or native?
native

What GPU are you using?
3090

**Screenshots/Config**
db_config.json
{
    "adam_beta1": 0.9,
    "adam_beta2": 0.999,
    "adam_epsilon": 1e-08,
    "adam_weight_decay": 0.01,
    "attention": "xformers",
    "center_crop": false,
    "concepts_path": "",
    "gradient_accumulation_steps": 1,
    "gradient_checkpointing": false,
    "half_model": false,
    "hflip": true,
    "learning_rate": 5e-07,
    "lr_scheduler": "constant",
    "lr_warmup_steps": 0,
    "max_grad_norm": 1,
    "max_token_length": 75,
    "max_train_steps": 10000,
    "mixed_precision": "fp16",
    "model_dir": "C:\\AI\\SDwebUI-AUTOMATIC1111\\models\\dreambooth\\guanyu9",
    "model_name": "guanyu9",
    "not_cache_latents": false,
    "num_train_epochs": 114,
    "pad_tokens": false,
    "pretrained_model_name_or_path": "C:\\AI\\SDwebUI-AUTOMATIC1111\\models\\dreambooth\\guanyu9\\working",
    "pretrained_vae_name_or_path": null,
    "prior_loss_weight": 1,
    "resolution": 512,
    "revision": 4000,
    "sample_batch_size": 1,
    "save_embedding_every": 1000,
    "save_preview_every": 1000,
    "scale_lr": false,
    "scheduler": "ddim",
    "src": "Bak\\r34f111sd15nd3.ckpt [f823ac0a]<div class='performance'><p class='time'>Time taken: <wbr>2m 58.87s</p><p class='vram'>Torch active/reserved: 7315/7394 MiB, <wbr>Sys VRAM: 16891/24576 MiB (68.73%)</p></div>",
    "train_batch_size": 1,
    "train_text_encoder": true,
    "use_8bit_adam": true,
    "use_concepts": false,
    "use_cpu": false,
    "use_ema": true,
    "concepts_list": [
        {
            "max_steps": -1,
            "instance_data_dir": "C:\\AI\\images\\guanyu\\guanyu_00FF00_WDN_output",
            "class_data_dir": "C:\\AI\\images\\regularization\\person_ddim",
            "file_prompt_contents": "Description",
            "instance_prompt": "[filewords]",
            "class_prompt": "man",
            "save_sample_prompt": "",
            "instance_token": "",
            "class_token": "",
            "num_class_images": 1500,
            "class_negative_prompt": "",
            "class_guidance_scale": 7.5,
            "class_infer_steps": 40,
            "save_sample_negative_prompt": "",
            "n_save_sample": 1,
            "sample_seed": -1,
            "save_guidance_scale": 7.5,
            "save_infer_steps": 40
        }
    ],
    "tokenizer_name": null
}

**Comment by d8ahazard** [comment]
Cache latents is enabled/disabled by toggling the "No Cache Latents" option in the "Parameters" section.

"With Prior Preservation" is enabled by setting the number of classifier images to greater than zero.

**Comment by OrochiZ** [comment]
当'Don't Cache Latents'没有被勾选并且'Total Number of Class/Reg Images'设置为1500时，'Cache Latents'阶段仍然没有出现。

**Comment by OrochiZ** [comment]
When 'Don't Cache Latents' is not checked and setting 'Total Number of Class/Reg Images' to 1500, the 'Cache Latents' stage still does not appear.
